src/features/Apiexplorer/Schema/RecursiveContent/RecursiveProperties/index.tsx

```diff
@@ -28,7 +38,17 @@ const RecursiveProperties = ({ is_open, properties, value }: TRecursivePropertie
             {index === 0 && value?.items?.description && (
               <SchemaDescription description={value.items.description} />
             )}
-            <SchemaObjectContent key={key} key_value={key} properties={properties} />
+            {key === 'forget_all' && 'oneOf' in value[key] ? (
+              <SchemaObjectContent
+                key={key}
+                key_value={key}
+                properties={properties}
+                jsonSchema={jsonSchema}
+                is_stream_types={true}
```

Comment:
can't we pass just  is_stream_types here

Reply:
Idts. `is_stream_types` default value is false, we explicitly pass true in this condition